content/en/docs/components/pipelines/sdk/output-viewer.md

```diff
@@ -318,6 +327,9 @@ curve CSV file as a string in `source` field directly.
 **Example:**
 
 ```Python
+def roc_vis(mlpipeline_ui_metadata_path: kfp.components.OutputPath()):
+  import json
+
   df_roc = pd.DataFrame({'fpr': fpr, 'tpr': tpr, 'thresholds': thresholds})
   roc_file = os.path.join(args.output, 'roc.csv')
```

Comment:
args.output is not defined, I think from context, it should be a gcs file prefix

Reply:
Sounds good, updated.